experimental/packages/opentelemetry-instrumentation/hook.mjs

```diff
@@ -1,12 +1,11 @@
-/*!
- * Copyright 2021 Datadog, Inc.
```

Comment:
I don't think this should be removed. Was that intentional?
Did the lint tooling perhaps change this? If so, we may need to look at fixing the lint tooling for this.

Reply:
Yeah that came from eslint. It looks like there's a `header/header` rule. I've reverted that part